README.md

```diff
@@ -1 +1,87 @@
-# updatecli-automation
\ No newline at end of file
+# Updatecli automation
+
+*Note:* This automation is still work in progress and subject to change.
+
+This project uses [Updatecli](https://github.com/updatecli/updatecli) to automate and orchestrate security related updates and versions bumps in Rancher owned projects.
+
+Initially, all the configuration files (manifests/pipelines) will reside inside this repo. In the future and if more teams decide to adopt the tool, we can move the pipelines files to their own respective target repos, like how Dependabot/Renovate works.
+
+## Tool
+
+We use Updatecli for this automation, instead of Dependabot or Renovate, because of its extensibility and multiple [plugins resources](https://www.updatecli.io/docs/prologue/introduction/) that allow greater flexibility when automating sequences of conditional update steps across multiple repos.
+
+For detailed information on how to use Updatecli, please consult its [documentation](https://www.updatecli.io/docs/prologue/introduction/) page.
+
+## Scope
+
+All Rancher owned repositories in [Rancher](https://github.com/rancher) and [Rancher Labs](https://github.com/rancherlabs) can be targeted by this automation.
+
+The main usage of Updatecli is for:
+
+* Bumping versions in unstructured formats, e.g., environment variables in Dockerfiles and by matching regular expressions.
+* Scripting the automation process, e.g., update package A in repo B after package X in repo Y matches a pre-defined version criteria.
+
+### Not in scope
+
+* Updatecli will only open a pull request in the targeted repo. It's not responsible for approving and merging the PR.
+* The resulting PR must still follow the rules of the targeted repo, e.g., passing checks, QA testing, review process etc.
+* Regular updates in major packages ecosystem, like Go Modules, should be done accordingly to the targeted repo owns definition, e.g., manually or by using Dependabot and/or Renovate.
+
+## Project organization
+
+A manifest or pipeline consists of three stages - source, condition and target - that define how to apply the update strategy.
+
+When adding a new manifest, please follow the example structure defined below.
+
+```
+.
+└── updatecli
+    ├── scripts                            # Contains the auxiliary scripts used in the manifests
+    ├── updatecli.d
+    │   └── rancher                        # Manifests targeting Rancher GH org
+    │       └── shell                      # Manifests to update rancher/shell repo
+    │           ├── prometheus-federator   # Manifests to update rancher/shell on a dependent repo
+    │           └── rancher
+    │               └── v2.6               # Manifests to update rancher/shell on a dependent repo - rancher - with multiple branches
+    │               └── v2.7
```

Comment:
This should be changed at some point, it seems like we need to duplicate the exact code/files per branch which leaves room to modify one and not the other.

Reply:
Similarly to Helm, Updatecli supports values and secrets(using mozilla/sops) files.

Reply:
I agree. There is room for future improvement in this part.

Reply:
Issue to track this improvement - https://github.com/rancherlabs/updatecli-automation/issues/3.